Association types - short update (#609)
akeneo/pim-helpcenter: content/md/first-steps/manage-type-association.md

1.  Only the labels can be edited for a type of association  
1.  Click on the `Save` button to save your changes

::: warning

> The code cannot be changed once the association type is created.  

Reviewer: ```suggestion
You cannot change the code once you created the association type.    
```

> Also, the 2-way association property cannot be updated.

Reviewer: ```suggestion
Neither will you be able to update the 2-way association. 
```
